spec/factories/navigation_links.rb

```diff
@@ -13,5 +13,11 @@
         navigation_link.class.set_callback :save, :before, :strip_local_hostname
       end
     end
+
+    trait :other_section_link do
+      after(:create) do |navigation_link|
+        navigation_link.update(section: "other")
+      end
```

Comment:
Let's maybe change this to the following:

```suggestion
      after(:build) do |navigation_link|
        navigation_link.section = "other"
      end
```

This gives us a bit more flexibility, e.g. we can now use `build(:navigation_link, :other_section_link)` to get a link in that `:other` section that's not persisted in the database, etc.

```ruby
nl = FactoryBot.build(:navigation_link, :other_section_link)
nl.persisted?
```